### DISCUSSION TITLE
Safari-Erweiterung nach jedem Safari-Neustart wieder weg

### DISCUSSION THREAD

**Discussion by heyklaus** [stefanw/voebbot#470] [Q&A]
Hallo – ich muss die Safari-Erweiterung nach jedem Safari-Neustart neu installieren und jedes Mal auch neu "Nicht signierte Erweiterungen erlauben" im Entwickler-Menü. Ist das so oder kann ich das irgendwie ändern?  Der regelmäßige Installationsvorgang, inklusive einloggen und Berechtigungen erteilen, ist doch etwas mühsam. Vielen Dank für sachdienliche Tipps und Hinweise!

**Comment by Maik-Wi** [comment]
Nein das kannst du nicht ändern, dazu müsste die Erweiterung im App Store gelistet sein, sie wird aber nicht durch den Review-Prozess von Apple kommen. Wechsle am besten einfach auf Firefox 😉
